src/app/globalStatusBar/userSetting/stub/logout.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @name logout
+ * @desc Log out the current user
+ * @param {Function} func - Function to execute after the promise resolves
+ * @returns {Promise} a fetch() resolving with whatever `func` returns
+ * @author minhducsun2002
+ */
+
+async function logout(func) {
+    return fetch(
+        `http://${window.location.hostname}:${window.location.port}/logout`
```

Comment:
Shorten this to "/api/logout"

Reply:
Address in https://github.com/vnma0/hestia/pull/19/commits/84c0282afababde0e3ad70cf90345d947d22553c.